### PR TITLE
1020 update landing pages for developers and designers getting started new

### PR DESCRIPTION
**Pull request by mi6-378**
## Summary of the changes

I have updated the Getting Started Pages, including adding cards and updating the 'Why we built a design system' page. I imported in the tree view branch and altered the nav priority of each page.

## Checklist

- [X] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

**Comment by GCHQ-Developer-112** [comment]
View your branch deployment here: https://mi6.github.io/ic-design-system-githubpages/branches/1020-update-landing-pages-for-developers-and-designers-getting-started-new

**Review by GCHQ-Developer-530 — CHANGES_REQUESTED** [review]
Can you update the links on the homepage to go to the design and development pages now that they exist? Should just be the href for 'Explore design' and the href for 'Explore development'

File: src/config.js [hunk 1]
```diff
@@ -107,8 +107,16 @@ module.exports = {
...
+          variant: "primary",
+        },
+        {
+          href: "/get-started/development",
+          text: "Read our design principles",
+          variant: "secondary",
+        },
       ],
     },
     develop: {
```
Comment (GCHQ-Developer-112) [inline src/config.js:122]: There are empty hrefs in this object. Was this intentional?

File: src/content/structured/get-started/index.mdx [hunk 2]
```diff
@@ -3,48 +3,314 @@ path: "/get-started"
...
+      />
+    </svg>
+  </IcCardHorizontal>
+</div>
 
-## Using v1.0 component library
+## Contribute
 
-You can still use v1.0 (link available for internal users only). Support will be limited and the library deprecated in the future.
+We are expanding the UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. If you would like to help, please [read more on how to contribute.](/community)
```
Comment (GCHQ-Developer-112) [inline src/content/structured/get-started/index.mdx:316]:
```suggestion
We are expanding the UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. If you would like to help, please read more on [how to contribute.](/community)
```

**Pull request closed without merging**